tests/help_generation_test.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test that we can generate help for PKB."""
+
+import os
+import unittest
+
+from perfkitbenchmarker import flags
+
+
+class HelpTest(unittest.TestCase):
+  def testHelp(self):
+    # Test that help generation finishes without errors
+    flags.GLOBAL_FLAGS.GetHelp()
+
+
+class HelpXMLTest(unittest.TestCase):
```

Comment:
I made them separate TestCase classes because I've been annoyed in the past that I can run a single TestCase class by itself, but not a single test method within that class. I'll change it if you still prefer having one class.